project.clj

```diff
@@ -5,7 +5,8 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [markdown-clj "0.9.63"]
-                 [me.raynes/fs "1.4.6"]]
+                 [me.raynes/fs "1.4.6"]
+                 [lein-light-nrepl]]
```

Comment:
So, check it: you'll want to add `lein-light-nrepl` to your own `~/.lein/profiles.clj` file -- since other people wont necessarily need it. It'll look something like:

```clojure
{:users {:dependencies [[lein-light-nrepl]]}
```

Also, this way it'll be globally available to all yer projects.